server/controllers/users.js

```diff
@@ -0,0 +1,87 @@
+import generateRandomString from "crypto-random-string";
+import database from "../models/index";
+import { getBaseUrl, httpResponse, serverError } from "../helpers/index";
+import { sendResetEmail } from "../emails/email";
+import userResponse from "../helpers/userResponse";
+
+const User = database["Users"];
+
+export const sendPasswordReset = async (req, res, next) => {
+  const userEmail = req.body.email;
+  try {
+    const verifiedUser = await User.findOne({
+      where: {
+        email: userEmail
+      }
```

Comment:
I thought we check a verified user by checking the `isVerified` field in the database. Using the email alone does not prove that the user is verified.